README.md

```diff
@@ -27,17 +27,16 @@ For details of the other Kata Containers repositories, see the
 * [Limitations](Limitations.md): differences and limitations compared with the default [Docker](https://www.docker.com/) runtime,
 [`runc`](https://github.com/opencontainers/runc).
 
-* HOWTO: [Kata Containers with k8s and cri-containerd](./how-to/how-to-use-k8s-with-cri-containerd-and-kata.md)
-* HOWTO: [OpenStack Zun with Kata Containers](./use-cases/zun_kata.md)
-* HOWTO: [Kata Containers with Firecracker](https://github.com/kata-containers/documentation/wiki/Initial-release-of-Kata-Containers-with-Firecracker-support)
-* HOWTO: [Sysctls with Kata Containers](./how-to/how-to-use-sysctls-with-kata.md)
-* HOWTO: [VM Templating](./how-to/what-is-vm-templating-and-how-do-I-use-it.md) and [VM Cache](./how-to/what-is-vm-cache-and-how-do-I-use-it.md)
+### Howto guides
+
+See the [howto documentation](how-to).
 
 ## Kata Use-Cases
 
-* [VPP with Kata](./use-cases/using-vpp-and-kata.md)
-* [SR-IOV with Kata](./use-cases/using-SRIOV-and-kata.md)
 * [GPU Passthrough with Kata](./use-cases/GPU-passthrough-and-Kata.md)
+* [OpenStack Zun with Kata Containers](./use-cases/zun_kata.md)
```

Comment:
I've also considered if use-cases and howtos should all be in one place - it's not really clear to me what the differentiation is, and sometimes I struggle to find docs due to the separation.

Reply:
Yes - the distinction is too subtle I think - let's just KISS and adopt one term consistently.